README.md

```diff
@@ -222,9 +222,15 @@ Examples
 
  - Build the kernel using a separate build directory, and run it, in verbose:
 ```
-   $ mkdir -p .virtme/build
```

Comment:
Is the directory creation no longer necessary?

Reply:
good point, I forgot to mention that in the commit message. I think after a recent change on the `virtme-configkernel` side, it is no longer needed to create this dir. I will update the commit message